changelog/fragments/update-k8s-controller-runtime-version.yaml

```diff
@@ -0,0 +1,31 @@
+# entries is a list of entries to include in
```

Comment:
None of these changes are user-facing, so I would remove this fragment.

Reply:
I agree with @estroz. 
The only entry that I can see that we must add is:  

- For Helm-based operators, upgrade Helm dependency from `v3.3.4` to `v3.4.1`

Reply:
I disagree, except for the helm dependency which has an [explicit sliding support window](https://helm.sh/docs/topics/version_skew/#supported-version-skew) (this is non-breaking for helm-operator because the v1.0 versions we officially support are within that window). There are no other changes `operator-sdk` users need to be aware of with these version bumps. Furthermore listing some of these changes can be confusing, ex. its easy to interpret "Operator SDK has bumped to k8s v1.19" as "my project can now support k8s v1.19" which isn't true.

Reply:
I think I can be OK with that.